.github/workflows/ci.yml

```diff
@@ -86,10 +86,10 @@ jobs:
       - uses: actions/checkout@v3.1.0
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.64.0
+          toolchain: stable
```

Comment:
> A better alternative (if it can be implemented) might be to use ${{ matrix.rust }} here as well and only enforce the check against the MSRV (since contributors shouldn't have to fix linting issues unrelated to their changes and those fixes should be in a dedicated commit).

Yes this is a better way.

Using `toolchain: stable` would update the used toolchain as soon as a new version is published (e.g. tomorrow 1.66.0 will be stable, in six more weeks it will be 1.67.0). That would mean that new lints are automatically "there" at some point and contributors would need to fix code that was working before and might not even be related to their contribution.

Thus having a fixed version is always better IMO.